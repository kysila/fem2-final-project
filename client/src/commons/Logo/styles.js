import { makeStyles } from '@material-ui/core';

export const useStyles = makeStyles((theme) => ({
  logo: {
    color: '#444444',
    margin: '0',
    alignItems: 'center',
    fontSize: '14px',
    outline: 'none',
  },
}));
