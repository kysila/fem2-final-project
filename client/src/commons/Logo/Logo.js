import React from 'react';
import Box from '@material-ui/core/Box';

import { Link } from 'react-router-dom';
import { useStyles } from './styles';

export function Logo() {
  const classes = useStyles();
  return (
    <Link to="/">
      <Box className={classes.logo}>
        <img src="/img/logo.svg" alt="Logo" />
      </Box>
    </Link>
  );
}
