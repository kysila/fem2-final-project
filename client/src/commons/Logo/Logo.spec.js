/* eslint-disable */
import React from 'react';
import { shallow } from 'enzyme';
import { Logo } from './Logo';

describe('Logo component', () => {
  it('Logo render', () => {
    const rendered = shallow(
      <Logo/>,
    );

    expect(rendered).toMatchSnapshot();
  });
});
