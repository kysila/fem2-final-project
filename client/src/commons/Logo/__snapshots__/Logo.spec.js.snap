// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo component Logo render 1`] = `
<Link
  to="/"
>
  <Styled(MuiBox)
    className="makeStyles-logo-1"
  >
    <img
      alt="Logo"
      src="/img/logo.svg"
    />
  </Styled(MuiBox)>
</Link>
`;
